assets/docs/vasudevgupta7/models/wav2vec2-960h/1.md

```diff
@@ -0,0 +1,34 @@
+# Module vasudevgupta7/wav2vec2-960h/1
+
+This model is fine-tuned on 960h of LibriSpeech dataset for Automatic Speech Recognition. For fine-tuning, base layer was initialized from pre-trained [wav2vec2](https://tfhub.dev/vasudevgupta7/wav2vec2/1)).
```

Comment:
You might want to add a note on the WER and why it's 3% higher than expected. Then you could add another note on how to get a better model that has 3% WER.